Content.Server/Cluwne/CluwneSystem.cs

```diff
@@ -45,13 +48,12 @@ public override void Initialize()
     private void OnMobState(EntityUid uid, CluwneComponent component, MobStateChangedEvent args)
     {
         if (args.NewMobState == MobState.Dead)
-		{
-            RemComp<CluwneComponent>(uid);
+        {
+            var damageSpec = new DamageSpecifier(_proto.Index<DamageGroupPrototype>("Genetic"), 300);
+            _damageable.TryChangeDamage(uid, damageSpec);
             RemComp<ClumsyComponent>(uid);
-            RemComp<AutoEmoteComponent>(uid);
```

Comment:
why you don't remove AutoEmote component, won't this make them giggle when dead?

Reply:
doesnt make em giggle when dead cause they require cluwnecomponent to giggle.